### DISCUSSION TITLE
Code completion for scripts not working in IntelliJ

### DISCUSSION THREAD

**Discussion by MarekZabka** [JetClient/jet-client-support#42] [Q&A] [answered]
I have installed the JetClient plugin in my IntelliJ.  The code completion for Pre-request and Test scripts does not work.  What should I do to enable it, please?

**Comment by Mahmoud-Elsaqqa** [comment]
install javascript and typescript intellij plugins

**Comment by AntonShuvaev** [comment, marked as answer]
@MarekZabka, sorry for the delayed reply. The code completion for Pre-request and Test scripts in the JetClient plugin depends on the JetBrains JavaScript plugin, which is available only in the paid versions of IntelliJ IDEs. Please ensure that this plugin is enabled if you are using a paid IDE. Unfortunately, this feature is not supported in free editions, such as IntelliJ IDEA Community Edition and Android Studio.

**Reply by MarekZabka** [reply]
Thank you, you are right!
